frontend/src/hooks/reportMapper/cycleTime.ts

```diff
@@ -6,16 +6,16 @@ export const cycleTimeMapper = ({
   swimlaneList,
   totalTimeForCards,
   averageCycleTimePerSP,
-  averageCircleTimePerCard,
+  averageCycleTimePerCard,
 }: CycleTimeResponse) => {
   const mappedCycleTimeValue: ReportDataWithTwoColumns[] = []
 
   const getSwimlaneByItemName = (itemName: string) => {
     return swimlaneList.find((item: Swimlane) => item.optionalItemName === itemName)
   }
-  const calPerColumnTotalTimeDivTotalTime = (itemName: string) => {
+  const calPerColumnTotalTimeDivTotalTime = (itemName: string): number[] | string[] => {
     const swimlane = getSwimlaneByItemName(itemName)
-    return swimlane ? [(parseFloat(swimlane.totalTime) / totalTimeForCards).toFixed(2)] : []
+    return swimlane ? [parseFloat((swimlane.totalTime / totalTimeForCards).toFixed(2))] : []
```

Comment:
for now, totalTime may be number type, should we still need parseFloat?

Reply:
The return value of `toFixed()` method  is `string`, so need to `parseFloat`.